docs/basics/sharing_pages.md

```diff
@@ -30,10 +30,11 @@ Once a page is initialized on SamePage, a status bar should appear beneath the p
 ![](/images/docs/basics/status.png)
 
 The icons after the SamePage logo from left to right are as follows:
-- `Invite Notebook` - This triggers the same modal from the initialization step for sharing a page with more notebooks.
-- `View History` - This opens a sidebar of all versions of your shared page, with clickable snapshots of the data.
-- `Disconnect Shared Page` - This will disconnect your notebook from the shared page. Note: this **does not delete** the page from being available on the decentralized web. It merely disconnects the page in _your_ notebook from receiving future updates. 
-- `Manual Sync` - In case there are any errors that occur with syncing data across notebooks, this icon is an escape hatch, allowing users to forcibly sync their version of the data
+
+- `Invite Notebook` – This triggers the same modal from the initialization step for sharing a page with more notebooks.
```

Comment:
majority of the doc changes you'll see are going from `-` to `–` after some `` `code` ``
It seems `markdown-to-jsx` doesn't support that.
I imagine there is a way to change what `ul` syntax is recognized, but I didn't see it.
![image](https://github.com/samepage-network/samepage.network/assets/3792666/c7b76548-3aeb-420a-a72f-a4472c94fc5f)